TESTING.md

```
@@ -0,0 +1,69 @@
+# Testing fetch_release_notes
+
+This document outlines the test scenarios for the `fetch_release_notes` function in `cat-launcher/src-tauri/src/fetch_releases/fetch_releases.rs`.
+
+## Scenarios
+
+### 1. Cache Hit with Body
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)